improver/standardise.py

```diff
@@ -236,6 +236,18 @@ def _discard_redundant_cell_methods(cube: Cube) -> None:
 
         cube.cell_methods = updated_cms
 
+    @staticmethod
+    def _remove_long_name_if_standard_name(cube: Cube) -> None:
+        """
+        Remove the long_name attribute from cubes if the cube also has a standard_name defined
+        """
+
+        if cube.standard_name and cube.long_name:
+            cube.long_name = None
+
+            print(cube)
+            return cube
```

Comment:
You are operating on the cube in place here, so you do not need a return statement.
This can be seen by the fact that you are not allocating what's returned into a new variable where you call it:
`self._remove_long_name_if_standard_name(cube)`
You would use a return statement for something like:
`result = self._remove_long_name_if_standard_name(cube)`

Reply:
thank you, I have changed the unit test to call process instead of the function now that the return statement is removed.